README.md

```diff
@@ -20,18 +20,19 @@ Using Docker is the recommended method of setting up an Avalon Media System Deve
 * ```cd avalon```
 * ```cp config/controlled_vocabulary.yml.example config/controlled_vocabulary.yml```
 * ```docker-compose pull```
-* ```docker-compose up avalon```
+* `docker-compose run createbuckets`
```

Comment:
triple back-ticks?

Reply:
Triple backticks are for multi-lines, the other lines didn't need it.